platform/ios/scripts/document.sh

```diff
@@ -32,6 +32,9 @@ mkdir -p ${OUTPUT}
 
 cp platform/ios/screenshot.png "${OUTPUT}"
 
+DEFAULT_THEME="platform/darwin/docs/theme"
+THEME=${JAZZY_THEME:-$DEFAULT_THEME}
```

Comment:
Interesting, I figured that the Makefile would still have to pass through this variable, just like it explicitly passes through `OUTPUT`. However, you’re right that the variable works regardless with both `make idocument JAZZY_THEME=…` and `JAZZY_THEME=… make idocument`.